packages/esm-service-queues-app/src/transition-latest-queue-entry/add-patient-to-queue-modal/add-patient-to-queue-entry.scss

```diff
@@ -0,0 +1,46 @@
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+
+.modalHeader {
+  :global {
+    .cds--modal-close-button {
+      position: relative;
+      inset-block-start: 0;
+      inset-inline-end: 0;
+      margin: 0;
+      margin-top: calc(-1 * #{layout.$spacing-05});
+    }
+
+    .cds--modal-close {
+      background-color: rgba(0, 0, 0, 0);
+
+      &:hover {
+        background-color: var(--cds-layer-hover);
+      }
+    }
+
+    .cds--popover--left > .cds--popover > .cds--popover-content {
+      transform: translate(-4rem, 0.85rem);
+    }
+
+    .cds--popover--left > .cds--popover > .cds--popover-caret {
+      transform: translate(-3.75rem, 1.25rem);
+    }
+  }
+}
+
+.container {
+  :global {
+    .cds--modal-content {
+      padding: 0 !important;
+    }
+  }
+}
```

Comment:
```suggestion
```
Removing the padding has made it it have no space on the bottom between the footer buttons and the last dropdown.
Before:
<img width="820" alt="Screenshot 2025-05-16 at 12 35 57 AM" src="https://github.com/user-attachments/assets/ec55ccce-7265-478a-859e-0186dafefe20" />
After:
<img width="820" alt="Screenshot 2025-05-16 at 12 39 41 AM" src="https://github.com/user-attachments/assets/9c420e13-6533-4a82-ad6a-b80965df6f65" />